### ISSUE TITLE
Mouse drag vs arrow key axis slider steps

### ISSUE THREAD

**Issue by d-v-b** [napari/napari#1572]
## 🐛 Bug

When drawing two 3D volumes with different voxel sizes, using arrow keys correctly increments the axis slider by the smaller of the two voxel sizes, but dragging the axis slider with the mouse does not.

## To Reproduce
```python
import numpy as np
import napari

scale = (.5,) * 3

with napari.gui_qt():
    data = np.random.randint(0,255, (10,10,10), dtype='uint8')
    viewer = napari.Viewer()
    viewer.add_image(data, name='raw')
    viewer.add_image(data, scale=scale, name=f'scaled_{scale}', colormap='blue')
```
on my mac laptop, the axis slider moves in increments of 2 when driven by the mouse, but increments of 1 when driven by arrow keys.

## Expected behavior

- mouse and keyboard slider interaction should have the same granularity
- mouse slider interaction should use the appropriate step size for the data

## Environment

napari: 0.3.7.dev30+gf101ef7
Platform: macOS-10.15.5-x86_64-i386-64bit
Python: 3.8.2 (default, May 6 2020, 02:49:43) [Clang 4.0.1 (tags/RELEASE_401/final)]
Qt: 5.12.5
PyQt5: 5.12.3
NumPy: 1.18.5
SciPy: 1.4.1
Dask: 2.20.0
VisPy: 0.6.4

GL version: 2.1 ATI-3.9.15
MAX_TEXTURE_SIZE: 16384

Plugins:
- napari-plugin-engine: 0.1.5
- svg: 0.1.3

**Comment by d-v-b** [comment]
Additional bug (I figured I'd skip making a second issue, since I suspect this is related): hitting the "play" button to view slices as an animation doesn't work when one of the datasets has been scaled. After hitting "play", nothing changes on screen until I pan the camera around, at which time the slices start getting animated. Makes me think something is queued to happen on redraw but not triggering redraw correctly. The step size of the animation is also wrong (too big, like the mouse drag step size)

**Comment by d-v-b** [comment]
looks like this gets fixed in #1539 (I had forgotten that PR was still open). I'll close this issue when that PR gets merged.

**Comment by jni** [comment]
Thanks @d-v-b! =) This is just the best kind of issue. 😂

**Comment by sofroniewn** [comment]
Ok great, @d-v-b thanks for reporting this and thanks for checking in #1539. I can look into this more too, but sounds like stuff that we will want to have fixed and tested in #1539!!!

**Comment by sofroniewn** [comment]
Closed by #1539
